app/views/shared/_accordion_group.html.haml

```diff
@@ -0,0 +1,25 @@
+-# Usage to include accordion group
+-# = render partial: 'shared/accordion_group', locals: { :title => 'Job Filters', items: [{ :title => 'locations' }, { :title => 'created by' }] }
```

Comment:
`render '/shared/accordion_group', title: 'Job filters', items: [{title: 'Locations', content: some_content_or_form_field}]`should work too